packages/react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -361,6 +361,7 @@ function useDismissHandlers(
             on(targetWindow, 'resize', dismissOnResize, true),
             on(targetWindow.document.documentElement, 'focus', dismissOnLostFocus, true),
             on(targetWindow.document.documentElement, 'click', dismissOnLostFocus, true),
+            on(targetWindow.document.documentElement, 'contextmenu', dismissOnLostFocus, true),
```

Comment:
I think a test needs to be added to ContextualMenu.test.tsx to ensure that the contextmenu event dismisses the menu.

Reply:
I added one, does not seem to fail on yarn test, what would be the best command to run  a specific one ?